phaser/column.py

```diff
@@ -91,6 +90,16 @@ def __init__(self,
         if self.null is False and self.default is not None:
             raise PhaserError(f"Column {self.name} defined to error on null values, but also provides a non-null default")
 
+    def _check_col_name(self, name):
+        if name is None or isinstance(name, float):
```

Comment:
What is the case when a float would be passed in?  And do we need to check for other numeric types, too, then?